cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -410,7 +410,7 @@ func (c *Client) addMember(name string, peerAddrs string, isLearner bool) ([]Mem
 		if isLearner {
 			// if learnerID is set, it means the etcd member is already added successfully.
 			if learnerID == 0 {
-				klog.V(1).Infof("[etcd] Adding etcd member as learner: %016x", peerAddrs)
+				klog.V(1).Infof("[etcd] Adding etcd member as learner")
```

Comment:
```suggestion
				klog.V(1).Infof("[etcd] Adding etcd member as learner: %s", peerAddrs)
```

Reply:
I wasn't sure that including the peer URLs would be helpful. If we really want to use the peer URLs, then we should also change the log message, e.g.

```suggestion
				klog.V(1).Infof("[etcd] Adding etcd member as learner (peer URLs: %s)", peerAddrs)
```

Reply:
It is better to include the `peerAddrs` information in the log for troubleshooting.
Would it be more consistent to replace `peer URLs` with `peer address`?

Reply:
I agree with you that we want to let the user know that we are

  (a) adding the etcd member using a set of peer addresses (or URLs), and
  (b) adding the etcd member as a learner

We do (b) in the log message being discussed here. And we do (a) here:

https://github.com/kubernetes/kubernetes/blob/afbedee92541bdd3d38d0d58b62ee6c704981438/cmd/kubeadm/app/phases/etcd/local.go#L155

Because both (a) and (b) have the same verbosity level, we know that, if user sees (b) they will see (b).

Therefore I think we can leave the log message discussed here as is.

If we change it, then the user will see peer addresses in both messages, which I don't think is helpful.

WDYT?

Reply:
Well, that seems to work too. But we should use `klog.V(1).Info` instead of `klog.V(1).Infof` here.

Reply:
Fixed. Because it was one-character change, I just amended my commit and force-pushed.